noisyatom/requirements.txt

```diff
@@ -1,5 +1,5 @@
 appdirs==1.4.0
-Django==1.10.5
+Django==2
```

Comment:
Shall we set this to 2.2 ? Your call. As long as it runs up the web server I don't care. Just let everyone know your decision. Thank you.

Reply:
Good call! I will change to Django==2.2